modules/cert-manager-enable-metrics.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-monitoring.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-enable-metrics_{context}"]
+= Enabling monitoring for the {cert-manager-operator}
+
+You can enable monitoring and metrics collection for the {cert-manager-operator} by using a service monitor to performing custom metrics scraping.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* The {cert-manager-operator} is installed.
+
+.Procedure
+
+. TODO: Step on verifying Prometheus selectors.
+
+. Add the label to enable cluster monitoring by running the following command:
++
+[source,terminal]
+----
+$ oc label namespace cert-manager openshift.io/cluster-monitoring=true
+----
+
+. Enable monitoring for user-defined projects. See _Enabling monitoring for user-defined projects_ for instructions.
+
+. Create a service monitor.
+
+.. Create a YAML file that defines the `ServiceMonitor` object:
++
+.Example `service-monitor.yaml` file
+[source,yaml]
+----
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: cert-manager
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/instance: cert-manager
+    app.kubernetes.io/name: cert-manager
+    app.kubernetes.io/version: v1.10.2
```

Comment:
```suggestion

```

I'd suggest to remove the version label from here. It would end up saving us some manual work of updating this yaml each release during docs update (as this absolutely does not alter any functionality).